glitter/blocks/form/models.py

```diff
@@ -14,6 +15,13 @@ class BaseFormBlock(BaseBlock):
     class Meta:
         abstract = True
 
+    def get_email_field(self, form):
```

Comment:
Maybe rename the method? It's not returning a field, it's more like `get_replyto_address`?